File: 2019/1/a.py
import fileinput
import numpy as np

def main():
  stdin = [line.rstrip() for line in fileinput.input()]

  input = np.array(stdin, dtype='int32')
  fuel = input / 3 - 2
  result = np.sum(fuel)

  print('%i' % result)

if __name__ == '__main__':
  main()

File: 2019/1/a.rs
use std::io::{self, Read};

fn main() -> io::Result<()> {
  let mut input = String::new();
  let stdin = io::stdin();
  let mut handle = stdin.lock();

  handle.read_to_string(&mut input)?;

  let list: Vec<i32> = input
    .split("\n")
    .map(|x| x.parse::<i32>().unwrap())
    .map(|x| x / 3 - 2)
    .collect();
  
  let mut sum: i32 = 0;
  for value in list {
    sum += value;
  }

  println!("{}", sum);

  Ok(())
}
